src/db/infusion/model.js

```diff
@@ -0,0 +1,17 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+
+const infusionSchema = new Schema({
+  startVolume: { type: String },
+  stopVolume: { type: String },
+  patientName: { type: String },
+  doctorsInstruction: { type: String },
+  status: { type: String },
```

Comment:
This `status` field should be an enum type of `active` and `ended`.

Reply:
fixed